### PR TITLE
PY3: handle basestring in a py2/py3 compatible way

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: binstar_build_client/worker/utils/script_generator.py [hunk 1]
```diff
@@ -66,7 +66,11 @@ def get_files(context, job_data):
     build_targets = job_data['build_item_info'].get('instructions', {}).get('build_targets')
     if not build_targets:
         return []
-
+    try:
+        unicode
```
Comment (srossross) [inline binstar_build_client/worker/utils/script_generator.py]: any reason not to test `basestring` directly?

Comment (srossross) [inline binstar_build_client/worker/utils/script_generator.py]: Otherwise this looks good to me.

Comment (ericdill) [inline binstar_build_client/worker/utils/script_generator.py]: There is no reason to test `basestring` directly.  However there is already this same try/except block on line 14 of this module. Do you have any objection to adding the `basestring = (str, bytes)` to that and getting rid of this try/except block here?

Comment (ericdill) [inline binstar_build_client/worker/utils/script_generator.py]: Err... there is no reason **not** to test `basestring` directly

**Pull request merged**